### PR TITLE
framework/media: Modify audio_manager

### PR DESCRIPTION
**Pull request by junmin-kim**

### CODE REVIEW

**Comment by seinfra** [comment]
Target : [e134540de2de07953de531ed0541ea90c57f466a] - Code Rule Check (C++) OK.

**Comment by seinfra** [comment]
Target : [e134540de2de07953de531ed0541ea90c57f466a] - Code Rule Check OK.

**Comment by seinfra** [comment]
Target : [242dda9344be644b5fbfdccde8a0ed2e03cac633] - Code Rule Check (C++) OK.

**Comment by seinfra** [comment]
Target : [242dda9344be644b5fbfdccde8a0ed2e03cac633] - Code Rule Check OK.

File: framework/src/media/audio/audio_manager.c [hunk 1]
```diff
@@ -1614,8 +1614,21 @@ audio_manager_result_t find_stream_in_device_with_process_type(device_process_ty
 	audio_card_info_t *card;
 
 	if (type == AUDIO_DEVICE_PROCESS_TYPE_NONE) {
-		meddbg("Type is AUDIO_DEVICE_PROCESS_TYPE_NONE\n");
-		return AUDIO_MANAGER_INVALID_PARAM;
+		medvdbg("Type is AUDIO_DEVICE_PROCESS_TYPE_NONE\n");
+		for (j = 0; j < CONFIG_AUDIO_MAX_DEVICE_NUM; j++) {
```
Comment (Taejun-Kwon) [inline framework/src/media/audio/audio_manager.c]: This is temp code, please put /* TODO will be implemented later based on audio subsystem's implemantation */ or some other comments.

Comment (junmin-kim) [reply framework/src/media/audio/audio_manager.c]: Added comments

**Comment by seinfra** [comment]
Target : [5418c9b056f611cb81e28d0506b9dee7375fab96] - Code Rule Check (C++) OK.

**Comment by seinfra** [comment]
Target : [5418c9b056f611cb81e28d0506b9dee7375fab96] - Code Rule Check OK.

File: framework/src/media/audio/audio_manager.c [hunk 2]
```diff
@@ -1614,8 +1614,22 @@ audio_manager_result_t find_stream_in_device_with_process_type(device_process_ty
 	audio_card_info_t *card;
 
 	if (type == AUDIO_DEVICE_PROCESS_TYPE_NONE) {
-		meddbg("Type is AUDIO_DEVICE_PROCESS_TYPE_NONE\n");
-		return AUDIO_MANAGER_INVALID_PARAM;
+		/* TODO : will be implemented later based on audio subsystem's implemantation */
```
Comment (jeongchanKim) [inline framework/src/media/audio/audio_manager.c]: please change implemantation to implementation

Comment (junmin-kim) [reply framework/src/media/audio/audio_manager.c]: Fixed it

**Comment by seinfra** [comment]
Target : [b87c17cb3b8c596e273ad1a6399cf7c64eaa8bea] - Code Rule Check (C++) OK.

**Comment by seinfra** [comment]
Target : [b87c17cb3b8c596e273ad1a6399cf7c64eaa8bea] - Code Rule Check OK.

**Pull request merged**